integration-tests/src/test/java/oracle/weblogic/kubernetes/ItWDTModelNoServer.java

```diff
@@ -0,0 +1,356 @@
+  private static final String clusterName = "cluster-1";
+
+  private static String domainNamespace = null;
+  private static String opNamespace = null;
+  private static LoggingFacade logger = null;
+  private static String imageName = null;
+
+  /**
+   * Get namespaces for operator and WebLogic domain.
+   *
+   * @param namespaces list of namespaces created by the IntegrationTestWatcher by the
+   *                   JUnit engine parameter resolution mechanism
+   */
+  @BeforeAll
+  public static void initAll(@Namespaces(2) List<String> namespaces) {
+
+    logger = getLogger();
+
+    // get a unique operator namespace
+    logger.info("Getting a unique namespace for operator");
+    assertNotNull(namespaces.get(0), "Namespace list is null");
+    opNamespace = namespaces.get(0);
+
+    // get a unique domain namespace
+    logger.info("Getting a unique namespace for WebLogic domain");
+    assertNotNull(namespaces.get(1), "Namespace list is null");
+    domainNamespace = namespaces.get(1);
+
+    // install and verify operator
+    installAndVerifyOperator(opNamespace, domainNamespace);
+  }
+
+  /**
+   * Test in the WDT model file there is no AdminServerName and no Server section defined.
+   * Verify that AdminServerName is set to 'AdminServer' and the admin server pod will be generated.
+   */
+  @Test
+  @DisplayName("Test in the WDT model file there is no AdminServerName and no Server section defined")
+  void testWdtModelNoAdminServerNameNoServer() {
+
+    String wdtModelFile = "model-noadminservername-noserver.yaml";
+    imageName = createAndVerifyDomainImage(wdtModelFile);
+
+    createMiiDomainAndVerify(domainNamespace, domainUid,
+        imageName, adminServerPodName, managedServerPrefix, replicaCount);
+
+    //check and wait for the application to be accessible in all server pods
+    for (int j = 1; j <= replicaCount; j++) {
+      String managedServerPodName = managedServerPrefix + j;
+      String expectedStr = "Hello World, you have reached server " + MANAGED_SERVER_NAME_BASE + j;
+
+      logger.info("Checking that application is running on managed server pod {0}  in namespace {1} with "
+          + "expectedString {3}", managedServerPodName, domainNamespace, expectedStr);
+      checkAppIsRunning(
+          domainNamespace,
+          managedServerPodName,
+          expectedStr);
+    }
+
+    logger.info("Domain {0} is fully started in namespace {1} - servers are running and application is available",
+        domainUid, domainNamespace);
+
+    // delete domain and cluster
+    shutdownDomainAndVerify(domainNamespace, domainUid, replicaCount);
+    deleteDomainResource(domainNamespace, domainUid);
+    deleteClusterCustomResource(clusterName, domainNamespace);
+  }
+
+  /**
+   * Test in the WDT model file AdminServerName is not specified and in Server section there is a server 'myadmin' set.
+   * Verify that AdminServerName is set to 'AdminServer' and the admin server pod with name domainuid-adminserver
```

Comment:
Is the myadmin server is an independent managed server ?

Reply:
Explicitly check the existence of AdminServer pod.

Reply:
yes, myadmin server is an independent managed server in this case.

Reply:
The pod domainuid-myadmin is also created --> A managed server pod domainuid-myadmin is also created

Reply:
updated the javadoc